README.md

```diff
@@ -1,16 +1,9 @@
-This tree contains BLHeli code for sensorless brushless motor electronic speed control (ESC) boards.  
-  
-To view and use the files, click the "Clone or download" button on this page,  
-and then select "Download ZIP" to download the repository to your computer.  
-  
-For flashing and configuration, download the BLHeliSuite PC software:  
-https://www.mediafire.com/folder/dx6kfaasyo24l/BLHeliSuite  
+This is a fork of the BLHeli\_S open source sensorless brushless motor electronic speed control (ESC) firmware.
+It was created in an attempt to revive open source development of this type of firmware, and make modern features available to users of open source software.
+ 
+For flashing and configuration, get the latest version of BLHeli Configurator from [here](https://github.com/blheli-configurator/blheli-configurator/releases).
+The latest releases of BLHeli\_S are downloadable from within the Configurator, or the hex files can be downloaded from [here](https://github.com/betaflight/BLHeli_S/releases).
   
 For more information, check out these threads:  
-https://www.rcgroups.com/forums/showthread.php?2640796 (for BLHeli_S)  
+https://www.rcgroups.com/forums/showthread.php?2640796 (for BLHeli\_S)  
```

Comment:
extra “\” ?

Reply:
`_` needs to be escaped with `\` in markdown.

Reply:
ah, ok